pom.xml

```diff
@@ -42,8 +41,7 @@
     <plugin.osmaven.version>1.6.2</plugin.osmaven.version>
     <plugin.signature.version>1.1</plugin.signature.version>
     <plugin.surfire.version>3.0.0-M3</plugin.surfire.version>
-    <!-- keep below 3.1.0 because 3.1.0 fails on java 8. -->
-    <plugin.javadoc.version>3.0.1</plugin.javadoc.version>
```

Comment:
The most recent plugin works well with java 8 and above, but reacts more harshly on certain javadoc errors, causing build to fail.